tests/serialization/test_dag_serialization.py

```diff
@@ -167,6 +167,7 @@
                     "max_retry_delay": 600.0,
                     "downstream_task_ids": [],
                     "_is_empty": False,
+                    "_is_skippable": False,
```

Comment:
Oh it just occurred to me that this name is wrong - it implies that this controls if this operator itself can be skipped, which isnt what this does.

How about `_can_skip_downstream`? Naming is hard